### PR TITLE
Fix inceptionv3

### PR DESCRIPTION
**Pull request by kevinthesun**
Fix bugs for conv2d padding and conversion of avg pooling from mxnet to nnvm.
@yzhliu @yidawang

### CODE REVIEW

**Comment by tqchen** [comment]
please https://docs.tvm.ai/contribute/code_review.html#ensure-test-coverage

**Comment by tqchen** [comment]
@merrymercy has a good point and may simplify the logic. Since this is a bugfix and the current way is also fine. I am going to merge this first, we can make followup changes

File: topi/python/topi/x86/conv2d_avx_1x1.py [hunk 1]
```diff
@@ -163,7 +163,7 @@ def _declaration_conv_NCHWc(wkl, sch, data, kernel):
     out_height = (wkl.height + 2 * HPAD - wkl.hkernel) // HSTR + 1
     out_width = (wkl.width + 2 * WPAD - wkl.wkernel) // WSTR + 1
 
-    DOPAD = (HPAD != 0 and WPAD != 0)
+    DOPAD = (HPAD != 0 or WPAD != 0)
```
[the remark this answers is not in the document]
Comment (yzhliu) [reply topi/python/topi/x86/conv2d_avx_1x1.py]: @merrymercy I remember inline padding will introduce if/else in conv computing, which actually slows down compare to doing copy beforehand.

[the remark this answers is not in the document]
Comment (merrymercy) [reply topi/python/topi/x86/conv2d_avx_1x1.py]: You can take a look at `nn.pad`. If all padding is zero, no if/else is introduced.
https://github.com/dmlc/tvm/blob/f32841fa839ecf09e512f702d3e9e28c27fe603f/topi/python/topi/nn/pad.py#L46

[the remark this answers is not in the document]
Comment (yzhliu) [reply topi/python/topi/x86/conv2d_avx_1x1.py]: oh, yes, you're saying `data_pad.inline()`, I thought you guys were talking about inline pad into conv.

**Pull request merged**